core/compute/compute.go

```diff
@@ -131,1039 +112,135 @@ func DoInstanceActivate(instance *model.Instance, host *model.Host, progress *pr
 	if startErr := dockerClient.ContainerStart(context.Background(), containerID, types.ContainerStartOptions{}); startErr != nil {
 		if created {
 			if err := utils.RemoveContainer(dockerClient, containerID); err != nil {
-				return errors.Wrap(err, "Failed to activate instance")
+				return errors.Wrap(err, constants.DoInstanceActivateError)
```

Comment:
The problem with this approach is later when debugging you won't know where the error came from except that it was this method.  Because this constance is used in multiple places.  It is better to not have a constants file and just put a unique error string describing what you were doing.  For example "removing previous container during start"